File: 프로그래머스/0/120883. 로그인 성공？/로그인 성공？.java
class Solution {
    public String solution(String[] id_pw, String[][] db) {
        String answer = "fail";
            for(int i =0; i<db.length; i++){
                if(id_pw[0].equals(db[i][0]) && id_pw[1].equals(db[i][1])){ //배열은 참조값! 따라서 값을 비교하고 싶으면 각 요소를 비교해야함
                    return "login";
                }
                else if(id_pw[0].equals(db[i][0])&& id_pw[1].equals(db[i][1]) == false){
                    answer = "wrong pw";
                }
            }
        return answer;
    }
}

File: 프로그래머스/0/120883. 로그인 성공？/로그인 성공？.py
def solution(id_pw, db):
    #id와 pw가 모두 일치하면 login
    #id 없음 -> fail
    #id만 일치 -> wrong pw
    if id_pw in db:
        return "login"

    for x in db:
        if id_pw[0] == x[0] and id_pw[1]!=x[1]:
            return "wrong pw"
        
    return "fail"